ddtrace/contrib/botocore/services/sqs.py

```diff
@@ -21,27 +20,25 @@
 
 log = get_logger(__name__)
 MAX_INJECTION_DATA_ATTRIBUTES = 10
+SERVICE_TYPES = {
+    # Use String since changing this to Binary would be a breaking change as other tracers expect this to be a String.
+    "sqs": "String",
+    # Use Binary since SNS subscription filter policies fail silently with JSON strings
+    # https://github.com/DataDog/datadog-lambda-js/pull/269 AWS will encode our value if it sees "Binary"
+    "sns": "Binary",
+}
 
 
-def _encode_data(trace_data):
-    """
-    This method exists solely to enable us to patch the value in tests, since
-    moto doesn't support auto-encoded SNS -> SQS as binary with RawDelivery enabled
-    """
-    return json.dumps(trace_data)
+def _encode_data(data):
+    # NB This method exists solely to enable us to patch the value in tests, since
+    # moto doesn't support auto-encoded SNS -> SQS as binary with RawDelivery enabled
+    return json.dumps(data)
```

Comment:
```suggestion
# NB This method exists solely to enable us to patch the value in tests, since
# moto doesn't support auto-encoded SNS -> SQS as binary with RawDelivery enabled
_encode_data = json.dumps
```

this is a pedantic suggestion, up to you whether you accept or not.